recipes-ni/ni-systemimage/files/nisystemimage

```diff
@@ -634,7 +641,32 @@ init_fd () {
 	init_fd_$MODE
 }
 
+init_fs_restoremode () {
+	# If the util did not previously exit cleanly, it's possible that the
+	# working mount points didn't get unmount them. To avoid double-mounts,
```

Comment:
Could you fixup the grammar in this comment you copied.

Reply:
Done